queries.sql

```diff
@@ -1,7 +1,18 @@
 -- Part 1: Test it with SQL
+id INT
+employer VARCHAR
+name VARCHAR
+skills VARCHAR
```

Comment:
```suggestion
-- id INT
-- employer VARCHAR
-- name VARCHAR
-- skills VARCHAR
```
Great work! You can keep these lines in comments. We just wanted to verify that you saw the structure of the inital table for `job`.